### PR TITLE
Select threshold from list

### PR DESCRIPTION
**Pull request by holisticode**
Closes https://github.com/ava-labs/avalanche-cli/issues/130

Instead of "When setting the number of control keys in deploy, we should add range parameters in the prompt", in this PR I am suggesting just showing a list to pick the threshold from.

### CODE REVIEW

File: cmd/subnetcmd/deploy.go [hunk 1]
```diff
@@ -171,7 +172,7 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 	var threshold uint32
 
 	if len(controlKeys) > 0 {
-		threshold, err = getThreshold(uint64(len(controlKeys)))
+		threshold, err = getThreshold(len(controlKeys))
```
Comment (felipemadero) [inline cmd/subnetcmd/deploy.go:175]: I am not completelly sure that len(controlKeys) should be the max. I think a max of somewhat 3 was refered to in some meeting.

Comment (holisticode) [reply cmd/subnetcmd/deploy.go:175]: I don't know of any such restriction, can you find the source please?

**Review by dasconnor — APPROVED** [review]

**Comment by dasconnor** [comment]
I think this is good for now. Later we might want to do a branching path. If there are a lot of control keys, it might break the interface. If `numControlKeys > 10` or something like that, we might want to capture via text rather than list.

File: cmd/subnetcmd/deploy.go (hunk 1 above)
Comment (holisticode) [reply cmd/subnetcmd/deploy.go:175]: Resolved as per https://github.com/ava-labs/avalanche-cli/pull/181#issuecomment-1216984871

**Pull request merged**